lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -22,7 +22,7 @@ def languages
         end
 
         def concurrency
-          config.fetch("concurrency", 2)
+          config.fetch("config", {}).fetch("concurrency", 2).to_i
```

Comment:
Not sure if we're using a late enough version of ruby, but you could use the `dig` method with default fallbacks here.
http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig

Certainly, not a merge blocker. I just like the `:dig` interface :wink: